crates/test_utils/wasm/wasm_workspace/validation_package_success/src/lib.rs

```diff
@@ -1,6 +1,57 @@
 use hdk3::prelude::*;
 
+const NUM_SONGS: usize = 30;
+
+#[hdk_entry(id = "song", required_validation_type = "element")]
+struct Song;
+
+#[hdk_entry(id = "artist", required_validation_type = "custom")]
+struct Artist;
+
+entry_defs![Song::entry_def(), Artist::entry_def()];
+
+#[hdk_extern]
+fn validation_package(
+    app_entry_type: AppEntryType,
+) -> ExternResult<ValidationPackageCallbackResult> {
+    let index = app_entry_type.id();
+    match u8::from(index) {
+        // Artist
+        1 => {
```

Comment:
Not related to this PR, but....Yuck. We need to provide a better way to match on entry type. Seems like the entry id should be opaque to devs, they should be able to match on the "artist" string in the `id` specified in `hdk_entry`. It's also confusing that they would specify the "id" as `"artist"`, but the actual "id" is `1`.

Reply:
Yeh it's really not ideal. The tricky part is enforcing unique strings